api/migrations/0001_initial.py

```diff
@@ -1,104 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Was the intent to delete this migration without deleting the second migration and without adding in the one your changes would generate?